src/Config.hs

```diff
@@ -146,6 +148,7 @@ castOptions opts = Options
   { oVaultHost      = oVaultHost opts
   , oVaultPort      = oVaultPort opts
   , oVaultAddr      = oVaultAddr opts
+  , oVaultAuthBackend = oVaultAuthBackend opts
```

Comment:
We should also print the value in the `Show` instance below; it helps to clarify which value was selected in the end when verbose logs are enabled.